src/Admin/Vue.php

```diff
@@ -27,26 +33,24 @@ public function init() {
 			$logged = $this->payplug_section_logged();
 			$payplug = (new PayplugGateway());
 
-			$payplug_wooc_settings = get_option( 'woocommerce_payplug_settings', [] );
-
-			if ((empty($payplug_wooc_settings['oney_thresholds_default_min'])) && (empty($payplug_wooc_settings['oney_thresholds_default_max']))) {
-				$payplug_wooc_settings['oney_thresholds_default_min'] = $payplug->min_oney_price;
-				$payplug_wooc_settings['oney_thresholds_default_max'] = $payplug->max_oney_price;
+			if ((empty($this->options['oney_thresholds_default_min'])) && (empty($this->options['oney_thresholds_default_max']))) {
+				$this->options['oney_thresholds_default_min'] = $payplug->min_oney_price;
+				$this->options['oney_thresholds_default_max'] = $payplug->max_oney_price;
 			}
 
-			unset($payplug_wooc_settings["payplug_live_key"]);
-			unset($payplug_wooc_settings["payplug_test_key"]);
-			unset($payplug_wooc_settings["payplug_password"]);
-			unset($payplug_wooc_settings["payplug_merchant_id"]);
+			unset($this->options["payplug_live_key"]);
+			unset($this->options["payplug_test_key"]);
+			unset($this->options["payplug_password"]);
+			unset($this->options["payplug_merchant_id"]);
 
 			return [
-				"payplug_wooc_settings" => $payplug_wooc_settings,
+				"payplug_wooc_settings" => $this->options,
 				"header"           		=> $header,
 				"login"     			=> $this->payplug_section_login(),
 				"logged"           		=> $logged,
-				"payment_methods"  		=> $this->payplug_section_payment_methods($payplug_wooc_settings),
-				"payment_paylater"  	=> $this->payplug_section_paylater($payplug_wooc_settings),
-				"status" 				=> $this->payplug_section_status($payplug_wooc_settings),
+				"payment_methods"  		=> $this->payplug_section_payment_methods($this->options),
+				"payment_paylater"  	=> $this->payplug_section_paylater($this->options),
```

Comment:
Same here, you don't need it as a parameter if it's property of the `Vue` class, am I right?